app/controllers/editions_controller.rb

```diff
@@ -109,6 +109,11 @@ def tagging
   def update_tagging
     Tagging::TaggingUpdateForm.new(params[:tagging_tagging_update_form]).publish!
     redirect_to :back, flash: { success: "Tags have been updated!" }
+  rescue GdsApi::HTTPConflict
+    redirect_to :back,
+    flash: {
+      danger: "Somebody changed the tags before you could. Your changes have not been saved."
```

Comment:
Is `redirect_to :back` reliable? Shouldn't we send them to a fixed page?

Also I'm wondering if we can make this more friendly to the user - it's almost like this error is taunting them... what should the user do next if this happens?